### PR TITLE
Create a Rack in an existing VPC

### PR DESCRIPTION
**Pull request by ddollar**
## Release Playbook
- [ ] Rebase against master
- [ ] Release branch ()
- [ ] Pass CI ()
- [ ] Code review
- [ ] Merge into master
- [ ] Release master ()
- [ ] Pass CI ()
- [ ] Update staging rack
- [ ] Edit [Rack release record](https://github.com/convox/rack/releases) and/or update [docs](https://github.com/convox/convox.github.io)
- [ ] Publish release
- [ ] Release CLI

### CODE REVIEW

File: cmd/convox/install.go [hunk 1]
```diff
@@ -217,48 +194,43 @@ func cmdInstall(c *cli.Context) error {
...
-	fmt.Println(Banner)
+	var subnet0CIDR, subnet1CIDR, subnet2CIDR string
 
-	distinctId, err := currentId()
-	if err != nil {
-		return stdcli.QOSEventSend("cli-install", distinctId, stdcli.QOSEventProperties{Error: err})
-	}
+	if cidrs := c.String("subnet-cidrs"); cidrs != "" {
+		parts := strings.SplitN(cidrs, ",", 3)
+		if len(parts) < 3 {
```
Comment (beedub) [inline cmd/convox/install.go]: should this be `!=`? as is, this would allow 4 or more cidrs

File: cmd/convox/install.go [hunk 2]
```diff
@@ -217,48 +194,43 @@ func cmdInstall(c *cli.Context) error {
...
-	} else if terminal.IsTerminal(int(os.Stdin.Fd())) {
-		fmt.Print("Email Address (optional, to receive project updates): ")
+	var subnetPrivate0CIDR, subnetPrivate1CIDR, subnetPrivate2CIDR string
 
-		email, err := reader.ReadString('\n')
-		if err != nil {
-			return stdcli.QOSEventSend("cli-install", distinctId, stdcli.QOSEventProperties{Error: err})
+	if cidrs := c.String("private-cidrs"); cidrs != "" {
+		parts := strings.SplitN(cidrs, ",", 3)
+		if len(parts) < 3 {
```
Comment (beedub) [inline cmd/convox/install.go]: same here

**Comment by ddollar** [comment]
Deleting a Rack with this new `api/dist/kernel.json` had a lot of dependency errors. Likely we need some more (or less?) `DependsOn` scattered throughout.

**Comment by ddollar** [comment]
Merged in #891

**Pull request closed without merging**